test/master-playlist-controller.test.js

```diff
@@ -101,7 +105,10 @@ QUnit.test('creates combined and audio only SegmentLoaders', function() {
               'created alternate audio track segment loader');
 });
 
-QUnit.test('if buffered, will request second segment byte range', function() {
+/* eslint-disable no-warning-comments */
+// TODO add back in once logic is returned in segment-loader
+/* eslint-enable no-warning-comments */
+QUnit.skip('if buffered, will request second segment byte range', function() {
```

Comment:
Should we still be skipping this one? Not that you have to add it back in right now, just curious.